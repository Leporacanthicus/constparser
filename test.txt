a=10;
b=a+2;
c=b+a-3;
