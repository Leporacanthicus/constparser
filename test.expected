val=10
val=12
val=19
val=-1
